src/darwin/CHIPTool/CHIPTool/View Controllers/QRCode/QRCodeViewController.m

```diff
@@ -478,8 +478,8 @@ - (void)setVendorIDOnAccessory
     }
 }
 
-// MARK: MTRDevicePairingDelegate
-- (void)onPairingComplete:(NSError * _Nullable)error
+// MARK: MTRDeviceControllerDelegate
+- (void)controller:(MTRDeviceController *)controller commissioningSessionEstablishmentDone:(NSError *)error
```

Comment:
This is missing the _Nullable annotationon the NSError, no?

Reply:
I added.